RedditSharp/Things/VotableThing.cs

```diff
@@ -158,5 +169,46 @@ public void ClearVote()
             var response = request.GetResponse();
             var data = WebAgent.GetResponseString(response.GetResponseStream());
         }
+
+        public void Report(ReportType reportType)
+        {
+            Report(reportType, "");
+        }
+
+        public void Report(ReportType reportType, string otherReason)
```

Comment:
Okay, wasn't paying much attention, just added it really quick and didn't see any default parameters in my open documents, so I changed it to be an overload. I resubmitted the fixes though.